src/modules/launcher/Plugins/Microsoft.Plugin.Program/Programs/UWP.cs

```diff
@@ -214,17 +214,5 @@ public enum PackageVersion
             Windows8,
             Unknown,
         }
-
-        [Flags]
-        [System.Diagnostics.CodeAnalysis.SuppressMessage("Naming", "CA1714:Flags enums should have plural names", Justification = "This name is consistent with the corresponding win32 flags: https://docs.microsoft.com/en-us/windows/win32/stg/stgm-constants ")]
```

Comment:
@htcfreek this is removing `stgm` -- apparently the last instance.

The spell checker is a bit picky, if you tell it that there's a `stgm` then it'll lump all the `Stgm` and `STGM`s together with it and ignore them all -- *as long as it finds an `stgm`*.

But, if you remove the last `stgm`, then it's like "welp, that `stgm` isn't here, better suggest not ignoring it, and by the way, you should ignore `SGTM` which I did find".

I'm very close to just reworking the spell checker to not do case folding. It'll simplify the code and mean I won't have to explain its logic...